### PR TITLE
Add script to run docker image for pipeline-tools

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# cd pipeline_dir
+# docker pull phenoscape/pipeline-tools:latest
+
+docker run -v "$(pwd)":/pipeline -ti phenoscape/pipeline-tools
```
Comment (hlapp) [inline run.sh:4]: This won't work. You'll have to have to default executable in the Dockerfile, or you must provide the program to run here. I.e., you can't "run" an image, there has to be program inside of the container that is to be run.

Comment (Shalsh23) [reply run.sh]: This was launching the bash shell in the container with the requisite environment, which I have been using to manually run the relevant `make` command. But yeah, I have now changed it to accept the command from the user on the command line.

Comment (hlapp) [reply run.sh]: Even if the intent is to run `bash`, it's poor practice to assume that that's what will happen. (If the Dockerfile changes, this assumption may quickly go to hell.)

**Comment by balhoff** [comment]
> So can you say why you have the `-t` and `-i` options? It sounds like you're going to pass the command to the script, which sounds like it will probably _not_ be an interactive bash shell. Are you going to be piping standard input into the container?

I don't expect we will be piping standard input to the container. While the command will frequently not be an interactive bash shell, it may be.

**Comment by balhoff** [comment]
By the way, this script is to be used for interactive(-ish) use of the tools provided by the container. It's not used by Jenkins.

**Comment by hlapp** [comment]
So it's entirely conceivable that you would run `./run.sh bash`? In that case it seems like `run.sh` is more like a macro that saves one from having to type the full `docker run` command line?

**Comment by Shalsh23** [comment]
> So it's entirely conceivable that you would run `./run.sh bash`? In that case it seems like `run.sh` is more like a macro that saves one from having to type the full `docker run` command line?

Yes, that is how atleast I was seeing it. More of a convenience script.

**Comment by balhoff** [comment]
> So it's entirely conceivable that you would run `./run.sh bash`? In that case it seems like `run.sh` is more like a macro that saves one from having to type the full `docker run` command line?

Absolutely! Sorry the use wasn't clear. It's basically just to save someone from remembering how to use `docker run`. @Shalsh23 could you just expand the usage comment in the script?

**Comment by hlapp** [comment]
I see. Wouldn't that suggest that the command be more appropriately named `run-container.sh` or something?

**Comment by balhoff** [comment]
> I see. Wouldn't that suggest that the command be more appropriately named `run-container.sh` or something?

Personally I prefer `run.sh`, as in `./run.sh make build/evolutionary-profiles.ttl`.

**Comment by hlapp** [comment]
What I'm saying is normally if a repo has `run.sh` in the root directory, the (usually justified) assumption will be that this needs to be executed, doesn't require parameters except when changing defaults, and will run whatever needs to be run in this repo.

This doesn't seem to be the case here. All I'm saying is why not be cognizant of naming conventions that are out there whether they match your personal preferences or not. There's a reason I was confused.

**Comment by balhoff** [comment]
I think this can be solved with appropriate documentation. @Shalsh23 please describe the purpose of `run.sh` in both the `README.md` and within `run.sh`.

**Pull request merged**